src/ScenarioManagerTreeList/ScenarioManagerTreeList.js

```diff
@@ -0,0 +1,126 @@
+// Copyright (c) Cosmo Tech.
+// Licensed under the MIT license.
+
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { useTranslation } from 'react-i18next';
+import { TextField, Typography } from '@material-ui/core';
+import '@nosferatu500/react-sortable-tree/style.css';
+import SortableTree from '@nosferatu500/react-sortable-tree';
+import { ScenarioUtils } from '@cosmotech/core';
+import ScenarioNode from './components';
+import useStyles from './style';
+
+const ScenarioManagerTreeList = (props) => {
+  const classes = useStyles();
+  const { t } = useTranslation();
+  const {
+    datasets,
+    scenarios,
+    moveScenario
+  } = props;
+
+  const [searchText, setSearchText] = useState('');
+  const [treeData, setTreeData] = useState(formatScenariosToRSTList(datasets, scenarios));
```

Comment:
Could be interesting to use "useMemo" hooks here:

`const rstList = useMemo(() => formatScenariosToRSTList(datasets, scenarios), [datasets,scenarios]);
const [treeData, setTreeData] = useState(rstList); `

and in `filterScenarios` function

`  if (!searchStr || searchStr.length === 0) {
      setTreeData(rstList);
      return;
    }`